Source/MQTTnet/Server/Internal/MqttClientStatistics.cs

```diff
@@ -73,7 +73,13 @@ public void HandleReceivedPacket(MqttPacket packet)
                 LastNonKeepAlivePacketReceivedTimestamp = LastPacketReceivedTimestamp;
             }
         }
-
+        public void ResetStatistics(){
+            Interlocked.Exchange(ref _sentApplicationMessagesCount,0);
+            Interlocked.Exchange(ref _receivedApplicationMessagesCount,0);
+            Interlocked.Exchange(ref _sentPacketsCount,0);
+            Interlocked.Exchange(ref _receivedPacketsCount,0);
```

Comment:
Should these be set to `1`?
I'm asking because of https://github.com/dotnet/MQTTnet/blob/e88b280eb0fac04267487555399bff7241af4349/Source/MQTTnet/Server/Internal/MqttClientStatistics.cs#L13-L17

BTW: there should be a space after the `,`.

Reply:
In my opinion 0 is fine because the user wants so to reset the entire statistics. The initial 1 is required to have correct values directly after start.

Reply:
Yeah as @chkr1011  said this is user reset.. 1 is only on init when client open connection...